webapp/src/components/CompetenciesView.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Box, Container, Grid, Button } from '@mui/material';
+
```

Comment:
Extra line?

Reply:
If you check other files, they separate component import from other imports. What do you think I should do? Should I be consistent with other two files I made changes? Or should I change other two files to be consistent with all the other files?